reference/constraints/UniqueEntity.rst

```diff
@@ -253,11 +253,98 @@ each with a single field.
 
 **type**: ``boolean`` **default**: ``true``
 
+``ignoreNull``
+~~~~~~~~~~~~~~
+
+**type**: ``bool``, ``string`` or ``array`` **default**: ``[]``
```

Comment:
```suggestion
**type**: ``bool`` | ``array`` | ``string`` **default**: ``[]``
```

Reply:
Vincent, thanks for reviewing this! I've just fixed the errors that you reported. Thanks.